src/inmanta/env.py

```diff
@@ -258,8 +258,12 @@ def install_from_list(self, requirements_list: List[str], detailed_cache: bool =
         """
             Install requirements from a list of requirement strings
         """
-        requirements_list = sorted(requirements_list)
+        # Install url-based requirements directly to prevent parsing issues
+        url_based_requirements = [r for r in requirements_list if "://" in r]
+        self._install(url_based_requirements)
```

Comment:
we shouldn't call _install twice, because 
1. it is slow
2. it potentially behaves different when calling it twice with half the list

When look at the surrounding code, there is a pre-existing problem:
1. prior to hashing, the list should be sorted

Reply:
Nice catch!